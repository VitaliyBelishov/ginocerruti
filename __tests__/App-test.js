/**
 * @format
 */

import 'react-native';
import React from 'react';
import Sizes from '../source/size/size-guide.screen.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const snap = renderer
    .create(<Sizes />)
    .toJSON();
  expect(snap).toMatchSnapshot();
});
