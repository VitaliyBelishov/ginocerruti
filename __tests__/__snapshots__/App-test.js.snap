// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    contentInsetAdjustmentBehavior="automatic"
    enableAutomaticScroll={true}
    enableOnAndroid={true}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={false}
    style={
      Object {
        "backgroundColor": "#fff",
        "marginHorizontal": 12,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={
          Object {
            "alignSelf": "flex-start",
            "borderBottomColor": "#ffd700cc",
            "borderBottomWidth": 1,
            "marginVertical": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 30,
              "fontWeight": "400",
              "marginBottom": 6,
            }
          }
        >
          Whats my size ?
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "marginTop": 12,
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 18,
            }
          }
        >
          Please select collection:
        </Text>
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderColor": "#e5e5e5",
              "borderRadius": 5,
              "borderWidth": 0.5,
              "flexDirection": "row",
              "justifyContent": "center",
              "marginTop": 8,
              "opacity": 1,
              "padding": 6,
              "paddingHorizontal": 32,
            }
          }
        >
          <Text
            style={
              Object {
                "fontSize": 14,
              }
            }
          >
            Evening Wear
          </Text>
        </View>
        <Image
          resizeMode="contain"
          source={
            Object {
              "testUri": "../../../source/src/image/size-guide.jpg",
            }
          }
          style={
            Object {
              "height": 750,
              "marginTop": 16,
              "width": 750,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignSelf": "flex-start",
            "borderBottomColor": "#ffd700cc",
            "borderBottomWidth": 1,
            "marginVertical": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 30,
              "fontWeight": "400",
              "marginBottom": 6,
            }
          }
        >
          Size calculator
        </Text>
      </View>
      <View
        style={Object {}}
      >
        <Text
          style={
            Object {
              "fontSize": 17,
            }
          }
        >
          Using size chart: Bridesmaids, Evening Wear, Prom Dresses, Lenovia Bridal, LENOVIA VIP
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginTop": 24,
            }
          }
        >
          <Text
            style={
              Object {
                "marginRight": 8,
              }
            }
          >
            cm
          </Text>
          <RCTSwitch
            accessibilityRole="button"
            onChange={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            onTintColor="#069C3B"
            style={
              Array [
                Object {
                  "height": 31,
                  "width": 51,
                },
                Array [
                  Object {
                    "marginRight": 8,
                  },
                  Object {
                    "backgroundColor": "#069C3B",
                    "borderRadius": 16,
                  },
                ],
              ]
            }
            thumbTintColor={null}
            tintColor="#069C3B"
            value={false}
          />
          <Text
            style={
              Object {
                "marginRight": 8,
              }
            }
          >
            inches
          </Text>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginTop": 12,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4c4c4c",
                  "fontSize": 12,
                }
              }
            >
              Bust
            </Text>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "borderColor": undefined,
                    "flexDirection": "row",
                    "minHeight": 42,
                    "paddingVertical": 10,
                  },
                  false,
                  undefined,
                ]
              }
            >
              <TextInput
                allowFontScaling={true}
                autoCapitalize="sentences"
                autoCorrect={true}
                autoFocus={false}
                editable={true}
                keyboardType="numeric"
                multiline={false}
                numberOfLines={1}
                onChangeText={[Function]}
                placeholder="Bust"
                rejectResponderTermination={true}
                returnKeyType="done"
                secureTextEntry={false}
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "paddingHorizontal": 8,
                      "paddingVertical": 0,
                    },
                    Object {
                      "borderColor": "#e0e0e0",
                      "borderWidth": 1,
                      "height": 42,
                      "marginVertical": 0,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "marginHorizontal": 8,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4c4c4c",
                  "fontSize": 12,
                }
              }
            >
              Weist
            </Text>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "borderColor": undefined,
                    "flexDirection": "row",
                    "minHeight": 42,
                    "paddingVertical": 10,
                  },
                  false,
                  undefined,
                ]
              }
            >
              <TextInput
                allowFontScaling={true}
                autoCapitalize="sentences"
                autoCorrect={true}
                autoFocus={false}
                editable={true}
                keyboardType="numeric"
                multiline={false}
                numberOfLines={1}
                onChangeText={[Function]}
                placeholder="Weist"
                rejectResponderTermination={true}
                returnKeyType="done"
                secureTextEntry={false}
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "paddingHorizontal": 8,
                      "paddingVertical": 0,
                    },
                    Object {
                      "borderColor": "#e0e0e0",
                      "borderWidth": 1,
                      "height": 42,
                      "marginVertical": 0,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4c4c4c",
                  "fontSize": 12,
                }
              }
            >
              Hips
            </Text>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "borderColor": undefined,
                    "flexDirection": "row",
                    "minHeight": 42,
                    "paddingVertical": 10,
                  },
                  false,
                  undefined,
                ]
              }
            >
              <TextInput
                allowFontScaling={true}
                autoCapitalize="sentences"
                autoCorrect={true}
                autoFocus={false}
                editable={true}
                keyboardType="numeric"
                multiline={false}
                numberOfLines={1}
                onChangeText={[Function]}
                placeholder="Hips"
                rejectResponderTermination={true}
                returnKeyType="done"
                secureTextEntry={false}
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "paddingHorizontal": 8,
                      "paddingVertical": 0,
                    },
                    Object {
                      "borderColor": "#e0e0e0",
                      "borderWidth": 1,
                      "height": 42,
                      "marginVertical": 0,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
          </View>
        </View>
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#069C3B",
              "borderRadius": 5,
              "flexDirection": "row",
              "justifyContent": "center",
              "marginTop": 8,
              "opacity": 1,
              "padding": 6,
              "paddingVertical": 16,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "fontSize": 15,
              }
            }
          >
            Calculate
          </Text>
        </View>
        <View
          style={
            Object {
              "alignSelf": "flex-start",
              "borderBottomColor": "#ffd700cc",
              "borderBottomWidth": 1,
              "marginVertical": 16,
            }
          }
        >
          <Text
            style={
              Object {
                "fontSize": 30,
                "fontWeight": "400",
                "marginBottom": 6,
              }
            }
          >
            Size Charts
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
